app-backend/app/src/main/scala/categories/Routes.scala

```diff
@@ -0,0 +1,69 @@
+package com.azavea.rf.toolcategory
+
+import scala.util.{Success, Failure}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import com.azavea.rf.auth.Authentication
+import com.azavea.rf.database.Database
+import com.azavea.rf.database.tables.ToolCategories
+import com.azavea.rf.datamodel._
+import com.azavea.rf.utils.UserErrorHandler
+import com.lonelyplanet.akka.http.extensions.PaginationDirectives
+
+trait ToolCategoryRoutes extends Authentication
+    with PaginationDirectives
+    with ToolCategoryQueryParametersDirective
+    with UserErrorHandler {
+  implicit def database: Database
+
+  // Not implementing an update function, since it's an emergency operation and should probably be done
+  // in the database directly to avoid orphaning categorized tools
```

Comment:
:+1: and we'll probably want to remove the `POST` endpoint entirely (or add authorization checks) eventually since right now any authenticated user can create a Category.